api/tests/views_tests.py

```diff
@@ -201,3 +214,166 @@ def test_post_relayaddress_flagged_error(free_user, free_api_client) -> None:
         "detail": "Your account is on pause.",
         "error_code": "account_is_paused",
     }
+
+
+class TermsAcceptedUserViewTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.path = "/api/v1/terms-accepted-user/"
+        self.fxa_verify_path = authentication.INTROSPECT_TOKEN_URL
+        self.uid = "relay-user-fxa-uid"
+
+    def _setup_client(self, token):
+        self.client = APIClient()
+        self.client.credentials(HTTP_AUTHORIZATION=f"Bearer {token}")
+
+    def tearDown(self):
+        cache.clear()
+
+    def test_no_authorization_header_returns_400(self):
+        client = APIClient()
+        response = client.post(self.path)
+
+        assert response.status_code == 400
+        assert response.json()["detail"] == "Missing Bearer header."
+
+    def test_no_token_returns_400(self):
+        client = APIClient()
+        client.credentials(HTTP_AUTHORIZATION="Bearer ")
+        response = client.post(self.path)
+
+        assert response.status_code == 400
+        assert response.json()["detail"] == "Missing FXA Token after 'Bearer'."
+
+    @responses.activate()
+    def test_invalid_bearer_token_error_from_fxa_returns_500_and_cache_returns_500(
+        self,
+    ):
+        fxa_response = _setup_fxa_response(401, {"error": "401"})
+        not_found_token = "not-found-123"
+        self._setup_client(not_found_token)
+
+        assert cache.get(get_cache_key(not_found_token)) is None
+
+        response = self.client.post(self.path)
+        assert response.status_code == 500
+        assert response.json()["detail"] == "Did not receive a 200 response from FXA."
+        assert responses.assert_call_count(self.fxa_verify_path, 1) is True
```

Comment:
question (blocking): In some phone tests, I used `assert rsp.call_count == 1` because it's less verbose. Is there a reason we need to use `assert_call_count` like this?

Reply:
😆 this is a following previous tests/copy-paste of code that was introduced in PR #1514 for `authentications_tests.py`. But looking at the `responses` [documentation](https://github.com/getsentry/responses/blob/master/README.rst#assert-based-on-the-exact-url), it also uses the `assert_call_count()` and `... is True`. I think using the library's recommendation is better.

Reply:
Cool, sounds good. Do you mind updating those phones tests too so we're consistent?